midterm/script.js

```diff
@@ -0,0 +1,83 @@
+(async() => {
```

Comment:
`async()`のカッコの前には半角スペースを1つ入れるのが一般的かな。

```suggestion
(async () => {
```